### DISCUSSION TITLE
Setting up Discord hooks on Synology

### DISCUSSION THREAD

**Discussion by Hecatonchires667** [lloesche/valheim-server-docker#534] [Q&A]
Ok... I am to dumb for this...

On Synology I can set up environment variables for the container.

I tried the discord hooks the same way but it didn't work out.

I tried to find an environment file to add the stuff for the discord messages but couldn't find it.

Can someone explain how to do this on the Synology maybe with a screenshot or two? That would be awesome 

I tried to add the variables in the docker config but this didn't work. :(

**Comment by Hecatonchires667** [comment]
> Ok... I am to dumb for this...
> 
> On Synology I can set up environment variables for the container.
> 
> I tried the discord hooks the same way but it didn't work out.
> 
> I tried to find an environment file to add the stuff for the discord messages but couldn't find it.
> 
> Can someone explain how to do this on the Synology maybe with a screenshot or two? That would be awesome 
> 
> I tried to add the variables in the docker config but this didn't work. :( 

Ok... Got it to work but i seemed to have filled the variables in the wrong order which causes $$SERVER_NAME to be empty when the server boots.. maybe i get it up and running... Shamefully you can't change order so I have to insert the bottom ones again and delete the other ones to get a proper order...

**Reply by malake** [reply]
Can you elaborate? I'm having an issue with this as well and I'm just now getting some time to work on it. I have loaded my variables into the Synology Container Environments Variable field and I can see them in the `env` if I open a shell into the container.
